saleor/core/templatetags/dashboard.py

```diff
@@ -1,8 +1,15 @@
 from django.template import Library
 
+from ...core.utils import get_country_name_by_code
+
 register = Library()
 
 
 @register.simple_tag
 def get_formset_form(formset, index):
     return formset.forms[index]
+
+
+@register.simple_tag
+def get_country_by_code(country_code):
```

Comment:
I think this can be done via `get_language_info` `i18n` templatetag

Reply:
`get_language_info` returns a list of tuples with *language* code and name (e. g. ('en', 'English')`), not *country* code and name (e. g. `('GB', 'United Kingdom')`).